manifests/server.pp

```diff
@@ -291,6 +291,8 @@
 #
 # $puppetserver_trusted_agents::       Certificate names of agents that are allowed to fetch *all* catalogs. Defaults to empty array
 #
+# $puppetserver_trusted_certificate_extensions:: An array of hashes of certificate extensions and values. Defaults to array.
```

Comment:
Is the `Defaults to array` really that useful? I see it's also for trusted_agents, but I have my doubts about the usefulness.

Reply:
I made it an array to allow full flexibility.  Not sure anyone really wants to do https://github.com/theforeman/puppet-puppet/pull/758/files#diff-60ae41fd0a31977447947f59940ee9a4R475 but if it was just a plain hash, that wouldn't be possible.

Reply:
I meant the literal `Defaults to array.` in the documentation string.

Reply:
oh... right.  Good point.